### PR TITLE
Created Readme

### PR DESCRIPTION
**Pull request by richarddushime**
I Added a comprehensive README file that includes an overview of the repository, usage instructions, and any other pertinent information that users may need.
Looking forward for a Review 
@dunxen @adamjonas @satsie @josibake  
I hope it s useful 
Thank you

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Welcome to the **BTrust Builders** GitHub repository for discussion questions related to the following courses. This repository is designed to facilitate meaningful discussions and enhance your understanding of key concepts.
+
+1. **Mastering Bitcoin - Discussion Questions**
+   - Explore and engage in discussions here : [Mastering Bitcoin Discussion Questions](https://github.com/richarddushime/discussion-questions/blob/main/mastering-bitcoin/README.md)
+
+2. **Mastering the Lightning Network - Discussion Questions**
+   - Explore and engage in discussions here: [Mastering the Lightning Network Discussion Questions](https://github.com/richarddushime/discussion-questions/blob/main/mastering-lightning/README.md)
+
+
+**NB:** An overview of the questions for which you will be the discussion leader will be shared over the usual channels. Daily discussions will be scheduled on the calendar.
```
Comment (satsie) [inline README.md:12]: What does NB mean?

Comment (satsie) [inline README.md:12]: Perhaps include a link to where people can learn more about discussions? If someone stumbles on this repo they may be interested in participating with a group.

**Comment by satsie** [comment]
This is great! Thank you for making this repo more user friendly, especially to folks that are new to GitHub :D 

Left a few comments. Nothing major. However I don't have merge privileges, and I'm certainly not the most qualified person to be reviewing this. Looks like @stickies-v was the last one to do some merging here so tagging him (no good deed goes unpunished :wink: )

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# BTrust Builders Discussion Questions Repository
+
+Welcome to the **BTrust Builders** GitHub repository for discussion questions related to the following courses. This repository is designed to facilitate meaningful discussions and enhance your understanding of key concepts.
+
+1. **Mastering Bitcoin - Discussion Questions**
+   - Explore and engage in discussions here : [Mastering Bitcoin Discussion Questions](https://github.com/richarddushime/discussion-questions/blob/main/mastering-bitcoin/README.md)
```
Comment (josibake) [inline README.md]:
```suggestion
   - Explore and engage in discussions here : [Mastering Bitcoin Discussion Questions](https://github.com/Qala-Dev/discussion-questions/blob/main/mastering-bitcoin/README.md)
   - ```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+2. **Mastering the Lightning Network - Discussion Questions**
+   - Explore and engage in discussions here: [Mastering the Lightning Network Discussion Questions](https://github.com/richarddushime/discussion-questions/blob/main/mastering-lightning/README.md)
```
Comment (josibake) [inline README.md]:
```suggestion
   - Explore and engage in discussions here: [Mastering the Lightning Network Discussion Questions](https://github.com/Qala-Dev/discussion-questions/blob/main/mastering-lightning/README.md)
 ```

**Review by josibake — COMMENTED** [review]
Thanks for adding this! The links still point to your fork, but once that is fixed this LGTM as a readme for the repo

File: README.md (hunk 1 above)
Comment (richarddushime) [reply README.md:12]: NB means : nota bene ( calling for a special attention)

Comment (richarddushime) [reply README.md:12]: the discussion groups are always shared  in the discord private channel  for fellows

File: README.md (hunk 3 above)
Comment (richarddushime) [reply README.md]: Thanks , made a new commit  with the changes 
@josibake

**Comment by josibake** [comment]
LGTM

**Pull request merged**